specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2025-03-01-preview/definitions.json

```diff
@@ -2264,6 +2264,23 @@
           "description": "Flag indicating whether SSL certificate name validation should be done when using self-signed certificates for this backend host.",
           "type": "boolean",
           "default": true
+        },
+        "serverCertificateThumbprints": {
```

Comment:
Can you please extend our samples to include at least one for both scenarios?

Here is an example of how I added examples for load balancer: https://github.com/Azure/azure-rest-api-specs/pull/36818